# Define the goal
float32 goal_angle  # Specify which dishwasher we want to use
---
# Define the result
float32 final_angle
---
# Define a feedback message 
float32 current_angle
